Automattic/wp-calypso
# Move FollowButton to blocks

The `FollowButton` currently lives in `client/components`, but it's better classified as a block. This PR moves the follow button to sit alongside the like button in `client/blocks`.

### Known issues

The component needs to be cleaned up and ES6-ified, but we'll do that in a followup PR.

### To test

Navigate through the Reader and ensure there's no follow button breakage as a result of this change.
